### DISCUSSION TITLE
Optimize Initial Lens Model for DESIJ0215-2909

### DISCUSSION THREAD

**Discussion by numjahan** [AstroBridge/BDLensing#76] [Q&A] [answered]
I changed the initial value of **'beta'** to **0.05**. Now, the residual looks like this. 
<img width="498" alt="Screenshot 2023-05-14 at 11 08 20 PM" src="https://github.com/ajshajib/BDLensing/assets/92924875/12313a36-629c-4b45-93d6-71f7757fb078">
The residual has improved slightly, but there is still a significant residual at the center. The reduced chi^2 value for this data is 1.8222465399310996. What could be causing the strong residual at the center, and how can I improve the model?
[Google Colab link to my notebook](https://colab.research.google.com/drive/11S1vWxr6nEfjPLmT68LVMllZqCy1lnfX?usp=sharing)

**Comment by ajshajib** [comment, marked as answer]
There can be two possibilities:

1. The bright blob at 1 o'clock can be a second source component whose counter image is the blue residual at 8 o'clock.
2. The bright bloc at 1 o'clock is a neighboring galaxy, and adding that to your model will improve the model.

You can try possibility 1 first by removing the mask on top of the 1 o'clock blob, then check if the shapelets want to create a set of two images at 1 and 8 o'clock.
